lib/iris/experimental/um.py

```diff
@@ -719,3 +720,142 @@ def close(self):
                     os.rename(tmp_file.name, self.filename)
             finally:
                 self._source.close()
+
+
+def cutout(ffv_src, filename, cutout_params):
+    """
+    Subset a UM FieldsFile variant to a specified i, j index and number of
+    points along the x and y axes.
+
+    Args:
+
+    * ffv_src (:class:`FieldsFileVariant`)
+        A source FieldsFileVariant to subset.
+    * filename (string)
+        The filename for the new result object (another FieldsFileVariant).
+    * cutout_params (list)
+        The cutout parameters : (i index, j index, number of x points, number
+        of y points).
+        The result has shape [number of x points, number of y points], starting
+        at position [i index, j index] in the source data.
+        The index values are zero-based.
+        An error is raised if the cutout exceeds the bounds of the source data.
+
+    Returns:
+
+        A new, open :class:`FieldsFileVariant` at 'filename', opened with
+        mode=:data:`~FieldsFileVariant.CREATE_MODE`.
+
+    .. note::
+
+        As the result is extracted according to cell indices, the total
+        resulting area covered is not consistent between data on different grid
+        positions, such as P, U and V grids.
+
+    .. note::
+
+        This routine cannot handle a variable grid, and will raise an error
+        if such a grid is present.
+
+    .. note::
+
+        This routine cannot fields with extension data, and will raise an error
```

Comment:
I do mean extra data.  Not well written either, will adjust..
It is a limitation of the routine : If LBEXT != 0 this affects LBLREC, and I wasn't totally clear that was an easy fix, so I just ruled it out for now.